helm_deploy/hmpps-prisoner-location/values.yaml

```diff
@@ -42,10 +42,10 @@ generic-service:
   namespace_secrets:
     hmpps-prisoner-location:
       APPINSIGHTS_INSTRUMENTATIONKEY: "APPINSIGHTS_INSTRUMENTATIONKEY"
-      API_CLIENT_ID: "API_CLIENT_ID"
-      API_CLIENT_SECRET: "API_CLIENT_SECRET"
-      SYSTEM_CLIENT_ID: "SYSTEM_CLIENT_ID"
-      SYSTEM_CLIENT_SECRET: "SYSTEM_CLIENT_SECRET"
+      AUTH_CODE_CLIENT_ID: "API_CLIENT_ID"
```

Comment:
need to leave these as `API_CLIENT_ID` as they are the keys we have in the secrets